src/averageAges.js

```diff
@@ -55,7 +77,31 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const targetPeople = people
+    .filter(person => person.mother)
```

Comment:
Consider using a more descriptive variable name instead of `targetPeople`. For example, `mothersWithSons`. 🤔